### ISSUE TITLE
Custom Identifier in Scan Report

### ISSUE THREAD

**Issue by ghost** [mercedes-benz/sechub#76]
My current project contains of multiple micro-services. To better distinguish where a security finding is located, I want to scan the micro-services individually and get a report for every micro-service separately. At the moment, it is possible to perform multiple scans on different source folders using the same SecHub project. Unfortunately the generated report does not contain any information, what has been scanned; despite the name of the report file, it is not possible to find out later, what has been scanned.

I would like to have the possibility to add a custom identifier into the generated report. This identifier could contain e.g. of
- A sub-project name (e.g. name of micro-service)
- A version number
- A git commit id
- ...

**Comment by de-jcup** [comment]
Hm.. interesting.

Maybe we should provide something inside configuration which can be full customized by build servers. 
When using sechub client you could also use variables inside your configuration.

An example proposal:
```json
{
        "apiVersion" : "1.0", 
        "reportIdentifier" : "{{ .YOUR_ENV_VARIABLE }}" ,
         ...
}
```
So inside API (json) its clear that this is a report identifier, set inside configuration and that it should appear inside reports. Providing a variable would be nice for builds - you could setup this by an environment variable as shown in example.

**Comment by Jeeppler** [comment]
Closed by: #2040.
